client/apps/edit/components/tests/edit_container.test.js

```diff
@@ -51,7 +52,16 @@ describe('EditContainer', () => {
       error: {},
       isSaved: false,
       saveArticleAction: jest.fn(),
-      toggleSpinnerAction: jest.fn()
+      toggleSpinnerAction: jest.fn(),
+      startEditingArticleAction: jest.fn(),
+      stopEditingArticleAction: jest.fn(),
+      updateArticleAction: jest.fn(),
+      user: {},
+      currentSession: {
```

Comment:
Just want to make sure I'm understanding correctly -- currentSession is the session logged from websockets?  This means it is not necessarily the session of the user on the line above?

Reply:
`currentSession` is the session of the user who is currently editing the article, so it's not necessarily the user authenticated in the browser session.